demos/http/http_demo_basic_tls/CMakeLists.txt

```diff
@@ -1,29 +1,32 @@
 set( DEMO_NAME "http_demo_basic_tls" )
+
+# Include HTTP library's source and header path variables.
+include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreHTTP/httpFilePaths.cmake )
+
 # Demo target.
 add_executable(${DEMO_NAME})
 
 target_sources(
     ${DEMO_NAME}
     PRIVATE
         "${DEMO_NAME}.c"
+        ${HTTP_SOURCES}
+        "${HTTP_INCLUDE_THIRD_PARTY_DIRS}/http_parser.c"
```

Comment:
I have merged you may change this to ${HTTP_THIRD_PARTY_SOURCES}

Reply:
Thanks.